src/eventDispatchers/imageRenderedEventDispatcher.js

```diff
@@ -16,14 +17,17 @@ const onImageRendered = function(evt) {
         tool.mode === 'enabled')
   );
 
-  const brushTools = toolsToRender.filter(
-    tool => tool instanceof BaseBrushTool
+  const brushAndSegmentationTools = toolsToRender.filter(
+    tool =>
+      tool instanceof BaseBrushTool || tool instanceof BaseSegmentationTool
```

Comment:
Have we landed on nixing the `BaseSegmentationTool`? Should I abstain from commenting on code that may be reverted?

This logic only cares that we have _any_ brush tools, right? Because they don't render, they just manipulate the shared data that the renderer uses?

Reply:
Hmm. Need to discuss this one, its a good point I hadn't considered.

Reply:
With the imminent nuking of `BaseSegmentationTool`, we still need a way to check if any segmentation tools are at least `passive`, as this indicates the label map data should be rendered. If all segmentation tools (including brushes) are `disabled`, the labelmap shouldn't be rendered, in the same way length measurements shouldn't render if a length measurement is absent.

How about either:
- A segmentation tool's constructor can add `this.isSegmentationTool = true`, so that we may filter to see if there are any segmentation tools.
- A segmentation mixin adds `this.isSegmentationTool = true` on the mixins initialisation step, along with `BaseBrushTool` setting `this.iSegmentationTool = true` in its constructor. All brushes are segmentation tools, all tools implementing a segmentation mixin are segmentation tools?